app/helpers/turbo_streams/flash_message_helper.rb

```diff
@@ -0,0 +1,11 @@
+module TurboStreams::FlashMessageHelper
+  def flash_message(flash)
+    messages = flash.map do |type, message|
+      turbo_stream_action_tag(:flash_message, type: type, message: message)
+    end.join.html_safe
```

Comment:
You can also use `.safe_join` https://api.rubyonrails.org/classes/ActionView/Helpers/OutputSafetyHelper.html#method-i-safe_join